client/state/happychat/selectors/has-happychat-localized-support.js

```diff
@@ -0,0 +1,21 @@
+/** @format **/
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import isHappychatClientConnected from 'state/happychat/selectors/is-happychat-client-connected';
+
+/**
+ * Returns true if Happychat client is connected and server is available to take new chats
```

Comment:
Shouldn't it say "... and server is available to take new localized chats"? This selector is only for localized support chats, right?

Reply:
That is correct, updated.